tools/travis/build.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -e
+
+# Build script for Travis-CI.
+
+SCRIPTDIR=$(cd $(dirname "$0") && pwd)
+ROOTDIR="$SCRIPTDIR/../.."
+WHISKDIR="$ROOTDIR/../openwhisk"
+
+# Install OpenWhisk
+
+cd $WHISKDIR
+tools/build/scanCode.py .
+
+cd $WHISKDIR/ansible
+
+ANSIBLE_CMD="ansible-playbook -i environments/travis"
+
+$ANSIBLE_CMD setup.yml
+$ANSIBLE_CMD prereq.yml
+$ANSIBLE_CMD couchdb.yml
+$ANSIBLE_CMD initdb.yml
+
+cd $WHISKDIR
+
+./gradlew distDocker
+
+cd $WHISKDIR/ansible
+
+$ANSIBLE_CMD openwhisk.yml
+
+# Set Environment
+export OPENWHISK_HOME=$WHISKDIR
+
+# Install Catalog
+
+cat $WHISKDIR/whisk.properties
+cd $ROOTDIR/packages
+./installCatalog.sh $WHISKDIR/config/keys/auth.whisk.system
+
+# Test
+cd $ROOTDIR
+./gradlew :tests:test
```

Comment:
@houshengbo Your code will:
1. remove the test codes under tests/src/packages.
2. copy the test codes from openwhisk-catalog to openwhisk.
3. invoke the tests of "package.*"

But I don't think I need to do that in travis CI, because
1. I have resolved the compiler dependencies so it's not needed to copy openwhisk-catalog/tests/src to openwhisk/test/src.
2. `./gradlew :tests:test` in openwhisk-catalog will only run the tests in openwhisk-catalog/tests/src. It will not run any tests in openwhisk/test/src. 

Please think about my points.

Daisy